src/nettyrpc.nim

```diff
@@ -40,28 +42,36 @@ proc sendall*(message: var NettyStream) =
       reactor.send(conn, d)
     message.clear()
 
-proc rpc*(conn: Connection, procName: string, vargs: tuple) =
+proc writeHashedProcName(procName: Strings) =
```

Comment:
Mark `{.inline.}` as it's a simple procedure, want to get it as much as possible inlined

Reply:
Ok, I will look up what `{.inline.}` does for future reference as well.

Reply:
It emits code with the `__inline` C convention, which tells the compiler "Hey if possible can you please try to inline this" It's not a garuentee but it increases the chance. Inlining meaning it copies the body instead of having a proc call.